Control coverage/Feature/AVDHostPool.md

```diff
@@ -156,3 +158,113 @@ properties.securityRules<br />
 
 ___ 
 
+## Azure_AVD_Audit_Enable_HostPool_Diagnostic_Settings
+ 
+
+### Display Name 
+Diagnostic logs must be enabled for AVD Host pool VMs.
+
+### Rationale 
+Logs should be retained for a long enough period so that activity trail can be recreated when investigations are required in the event of an incident or a compromise. A period of 1 year is typical for several compliance requirements as well.
+
+
+### Control Settings 
+```json 
+{
+    "DiagnosticForeverRetentionValue": "0",
+    "DiagnosticMinRetentionPeriod": "365",
+    "DiagnosticLogs": [
+        "Checkpoint",
+        "Error",
+        "Management"
+    ]
+}
+ ```  
+
+### Control Spec 
+
+> **Passed:** 
+> Diagnostic settings should meet the following conditions:
+>   1. Diagnostic logs are enabled.
+>   2. At least one of the below logs destination is configured:
+>       a. Log Analytics.
+>       b. Storage account with min Retention period of 365 or forever(Retention period 0).
+>       c. Event Hub.
+> 
+> **Failed:** 
+> If any of the below conditions are meet:
+>   1. Diagnostic settings should meet the following conditions:
+>       a. All diagnostic logs are not enabled.
+>       b. No logs destination is configured:
+>          i. Log Analytics.
+>          ii. Storage account (with min Retention period of 365 or forever(Retention period 0).
+>          iii. Event Hub.
+>   2. Diagnostics settings is disabled for resource.
+
+ 
+### Recommendation 
+
+- **Azure Portal** 
+    - You can change the diagnostic settings from the Azure Portal by following the steps given here: https://docs.microsoft.com/en-us/azure/azure-monitor/essentials/diagnostic-settings
+      
+
+### Azure Policies or REST APIs used for evaluation 
+
+- REST API used to list diagnostic settings and its related properties at Resource level:
+/{ResourceId}/providers/microsoft.insights/diagnosticSettings?api-version=2021-05-01-preview<br />
+**Properties:**
+properties.metrics.category,properties.metrics.enabled,properties.metrics.retentionPolicy.enabled, properties.metrics.retentionPolicy.days
+properties.logs.category, properties.logs.categorygroup,properties.logs.enabled,properties.metrics.logs.enabled, properties.logs.retentionPolicy.days, name, properties.workspaceId,properties.storageAccountId,properties.eventHubName
+
+- REST API used to list diagnostic category group mapping and its related properties at Resource level:
+/{ResourceId}/providers/Microsoft.Insights/diagnosticSettingsCategories?api-version=2021-05-01-preview <br />
+**Properties:**
+properties.categoryGroups, name
+<br />
+___ 
+
+
+
+
+## Azure_AVD_NetSec_Restrict_Public_IPs
+ 
+
+### Display Name 
+Public IPs must not be open on AVD Host pool VMs.
+
+### Rationale 
+Public IPs provide direct access over the internet exposing the VM to attacks over the public network. Hence AVD Host pool VMs must not be accessible to any public IPs.
+
+
+### Control Spec 
+
+> **Passed:** 
+>   Public IPs are disabled on Azure AVD host pool VMs.
```

Comment:
if space has been added in the passed line, it shud be there in the failed also.

Reply:
Fixed.

Reply:
Although on the preview side it looks exactly same.